content/en/docs/zero-code/java/spring-boot-starter/annotations.md

```diff
@@ -120,3 +120,9 @@ annotation:
 | Name    | Type     | Description    | Default Value         |
 | ------- | -------- | -------------- | --------------------- |
 | `value` | `String` | Attribute name | Method parameter name |
+
+## Next steps
+
+Beyond the use of annotations, the OpenTelemetry API allows you to obtain a
+tracer that can be used for [custom instrumentation](../api) and execute code
+within the scope of that span.
```

Comment:
I didn't follow what "executing code within the scope of that span" meant, maybe reword or remove?

```suggestion
Beyond the use of annotations, the OpenTelemetry API allows you to obtain a
tracer that can be used [custom instrumentation](../api).
```